packages/util/src/stateful.ts

```diff
@@ -39,7 +39,7 @@ export class StateObject<U, I> {
     protected set<K extends keyof U>(key: K, newVal: U[K], batchMode?: boolean): IEvent;
     protected set<K extends keyof U>(keyOrNewVals: K | U, newVal?: U[K], batchMode: boolean = false): IEvent[] | IEvent | null {
         if (typeof keyOrNewVals === "string") {
-            return this.setSingle(keyOrNewVals, newVal as U[K], batchMode);
+            return this.setSingle(keyOrNewVals as any, newVal as U[K], batchMode);  //  TODO:  "as any" should not be needed (TS >= 3.1.x)
```

Comment:
Why is "as any" needed here if the version is being bumped to `3.1.4`?

Reply:
The version bump caused the new syntax error - and that works around it (for now).